src/screens/Main.tsx

```diff
@@ -37,22 +48,49 @@ const ActionComponents = {
 };
 
 function Main() {
-  const { actions, action, setAction } = useGUIContext();
+  const classes = useStyles();
+  const { actions, action, setAction, isBridged, setBridged } = useGUIContext();
   const searchItems = (choice: TransactionTypes) => actions.find((x: MenuActionItemsProps) => x.type === choice);
 
+  const handleOnSwitch = (event: React.MouseEvent<HTMLElement>, newAlignment: string | null) => {
+    setBridged(Boolean(newAlignment));
+  };
+
+  // To remove this check once the whole functionality is ready.
+  const isDev = process.env.REACT_APP_IS_DEVELOPMENT === 'true';
```

Comment:
Why? We don't have a production/stable deployment, it's fine to see this code and test it on our current test deployment.

Reply:
OK!